src/any_agent/tools/mcp/frameworks/agno.py

```diff
@@ -15,44 +23,39 @@
     mcp_available = True
 
 
-class AgnoMCPServerBase(MCPServerBase, ABC):
-    server: Any | None = None  # Using `Any` to avoid circular import issues
-    framework: Literal[AgentFramework.AGNO] = AgentFramework.AGNO
-
-    def _check_dependencies(self) -> None:
-        """Check if the required dependencies for the MCP server are available."""
-        self.libraries = "any-agent[mcp,agno]"
-        self.mcp_available = mcp_available
-        super()._check_dependencies()
+class AgnoMCPConnection(MCPConnection, ABC):
+    _server: AgnoMCPTools | None = PrivateAttr(default=None)
```

Comment:
This change would undo my bugfix from https://github.com/mozilla-ai/any-agent/pull/195/files: I used the type of Any because otherwise this code will throw an error if Agno isn't installed, even if the agno agent isn't being used. I also didn't use the TYPE_CHECKING blog because doing so was throwing pydantic circular import issues.

Reply:
This comment applies to the use of "Any" in all of the MCP framework classes.

Reply:
I took a different approach and use strings for the typing, that allows to keep the original type, avoid circular imports and prevent the missing import error

Checked with the uv sync command from #195